src/mixins/datetime.js

```diff
@@ -92,7 +92,11 @@ export default {
          */
         datetimeFormat(value, format) {
             if (value !== undefined && value !== "") {
-                return dayjs.utc(value).tz(this.timezone).format(format);
+                const date = new Date(value);
+                return date.toLocaleString(this.$i18n.locale, { ...format,
```

Comment:
You are disregarding the time zone here.

This post may clear up some confusion: https://stackoverflow.com/questions/15141762/how-to-initialize-a-javascript-date-to-a-particular-time-zone